cache/mock/cache.go

```diff
@@ -1,7 +1,7 @@
 package mocks
 
-import "github.com/go-gitea/lgtm/cache"
-import "github.com/stretchr/testify/mock"
+import cache "github.com/go-gitea/lgtm/cache"
```

Comment:
Ok. Thanks. This doesn't break anything. So I'm going to merge.
Really good job 👍